src/api.js

```diff
@@ -20,6 +22,14 @@ export function voteOnArticle(id) {
   return `${Api.articles}/${id}`;
 }
 
+export function postComment(id) {
+  return `${Api.articles}/${id}/comments`;
+}
+
+// export const axiosApiServer = axios.create({
+//   baseURL: 'https://nc-news-be-proj.onrender.com/api/articles',
+// });
```

Comment:
I would get rid of your commented out code when finished - although axios.create is a nice idea!